### DISCUSSION TITLE
Hileli Paralar

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#50] [Zeka Sorusu]
Elimizde görünüşleri aynı dokuz adet madeni para var. Bunların sadece biri normal ve sekizi sahte. Sahte paraların dördü aynı ağırlıkta ve normal paradan hafif, diğer dört sahte para da aynı ağırlıkta ve normal paradan ağır. Normal parayı çift kefeli bir teraziyle kaç tartıda bulabiliriz?

**Comment by Berkek007** [comment]
Elimizde 4 hafif, 4 ağır ve 1 normal (yani kendine özel ağırlığı olan) toplam 9 madeni para var. Bunları üçerli şekilde gruplarsak 3 grup elde ederiz. Grup1 ve Grup2 ardından Grup2 ve Grup3 teraziyle tartılacaktır. Eğer Gruplardan ikisi eşit çıkarsa, örneğin grup 1 ve 2 o zaman normalin grup3te olduğunu anlarız ve o gruptaki paraları 1. ve 2. sonra 2. ve 3. şeklinde tartarız eğer 1 ve 2 eşitse 3 normal. 2 ve 3 eşitse 1 normal eşitlik yoksa 2 normaldir yani ilk başta 2 grup tartışı sonra da 2 tartış yani 4 tartış ile bulma imkanımız var.

4

**Reply by sonsuzus** [reply]
İlk tarttığın grupların eşit çıkması mümkün mü?

Yani hafiflere sırayla h1,h2,h3,h4 diyelim
Ağırlara da a1,a2,a3,a4

diyelim.

Nasıl olursa eşit olur?

**Reply by sonsuzus** [reply]
Tamam şöyle olabilir doğru. 
a1,a2,h1 --- a3,a4,h2
